go.mod

```diff
@@ -7,12 +7,11 @@ require (
 	github.com/google/go-cmp v0.5.5
 	go.uber.org/zap v1.16.0
 	golang.org/x/sync v0.0.0-20201207232520-09787c993a3a
-	istio.io/api v0.0.0-20201123152548-197f11e4ea09
+	istio.io/api v0.0.0-20210322145030-ec7ef4cd6eaf
 	istio.io/client-go v1.8.1
-	istio.io/gogo-genproto v0.0.0-20191029161641-f7d19ec0141d // indirect
-	k8s.io/api v0.19.7
-	k8s.io/apimachinery v0.19.7
-	k8s.io/client-go v0.19.7
+	k8s.io/api v0.20.5
```

Comment:
Just saw this PR. I am wondering if we want to raising the k8s version as it will bump the min k8s version to be supported in the Knative 0.22 release.

Let me double check.

Reply:
OK. per https://github.com/knative/community/blob/main/mechanics/RELEASE-VERSIONING-PRINCIPLES.md#knative-serving-version-table, there is no plan to bump k8s version for knative 0.22. We need to revert the k8s related changes.